openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -187,7 +187,7 @@ def _submit_deadline_post_job(self, instance, job, instances):
             create_metadata_path(instance, anatomy)
 
         environment = {
-            "AVALON_DB": os.environ["AVALON_DB"],
+            "AVALON_DB": os.getenv("AVALON_DB"),
```

Comment:
Should be on line 206.

Reply:
done in https://github.com/ynput/OpenPype/pull/6263/commits/b5633afffabe5e2a8143e5a80371db2217c73683